nilearn/decoding/searchlight.py

```diff
@@ -191,9 +192,21 @@ def _group_iter_search_light(
         if isinstance(cv, KFold):
             kwargs = {"scoring": scoring}
 
-        par_scores[i] = np.mean(
-            cross_val_score(estimator, X[:, row], y, cv=cv, n_jobs=1, **kwargs)
-        )
+        if y is None:
```

Comment:
Can you update the doc string above (line 158) to mention that y can be None and explain what happens in this case?

Reply:
Done. Please check.